src/Storefront/Resources/theme.json

```diff
@@ -29,219 +29,113 @@
     "app/storefront/dist/assets"
   ],
   "config": {
-    "blocks": {
```

Comment:
Can it be that plugins use this theme.json for own reasons and will run into issues due this removal?

Reply:
Nope. It's on a similar level like composer.json or package.json files.

But even when they're using that, the new system will catch this.